cmd/minikube/cmd/tunnel.go

```diff
@@ -116,6 +122,33 @@ var tunnelCmd = &cobra.Command{
 	},
 }
 
+func cleanupLock() {
+	if lockHandle != nil {
+		lockHandle.Unlock()
+	}
+}
+
+func checkNoOtherTunnelProcess() {
+	tunnelLockPath := filepath.Join(localpath.MiniPath(), ".tunnel_lock")
+
+	_, err := os.OpenFile(tunnelLockPath, os.O_RDWR|os.O_TRUNC, 0600)
+	if err != nil {
+		if os.IsNotExist(err) {
+			_, err = os.Create(tunnelLockPath)
+			if err != nil {
+				exit.Error(reason.SvcTunnelStart, fmt.Sprintf("error creating lock for tunnel file (%s)", tunnelLockPath), err)
```

Comment:
Lets create a new reason in reason package and we can call it
TUNNEL_ALREADY_RUNNING

also make sure the exit message does not have github box since this is not a crash or issue. This is a usage problem.

look at minikube profile list code, when there is no cluster running it exits nicely (no need to show github issue box and crying emoji)

Reply:
@medyagh 
1. I've added the new reason as proposed.
2. Fixed the github box, however I've made this change only when we fail to _acquire_ filelock. If there are errors in creating the file itself or creating the lock handle itself, then I have left it as is (with github box). Let me know if this is not the correct thing to do. (Note: I referred to `service list` code instead of `profile list`. I couldn't seem to find the example in `profile list`)